### PR TITLE
feat: add variant prop to `Input`

### PR DESCRIPTION
**Pull request by Sandtory**
- move props `label`,  `meta` and `helperText` from `Input` to `TextField`
- add `variant `prop to Input component
- add `helperIcon` and `inputIcon` to TextField

### CODE REVIEW

File: packages/components/src/components/TextField/TextField.tsx [hunk 1]
```diff
@@ -4,38 +4,95 @@ import React, { forwardRef } from "react";
...
-                unit && (
-                    <View style={styles.unit}>
-                        <Typography variant="label" color="textTertiary">
-                            {unit}
-                        </Typography>
+export const TextField = forwardRef<TextInput, TextFieldProps>(
+    ({ unit, helperText, label, meta, helperIcon, inputIcon, ...rest }, ref) => {
+        const styles = useStyles(themeStyles, { variant: rest.variant });
+        return (
+            <View style={{ flexGrow: 1 }}>
```
Comment (Sandtory) [inline packages/components/src/components/TextField/TextField.tsx:46]: Added flexGrow in order to make it more versatile when used in other projects and in a flex grid 🤠

File: packages/components/src/components/TextField/TextField.tsx [hunk 2]
```diff
@@ -4,38 +4,95 @@ import React, { forwardRef } from "react";
...
+        alignItems: "center",
+    },
+    label: {
+        paddingHorizontal: theme.spacing.textField.paddingHorizontal,
+        color: props.variant ? theme.colors.interactive[props.variant] : theme.colors.textTeritary,
+    },
+    iconColor: {
+        color: props.variant ? theme.colors.interactive[props.variant] : theme.colors.textPrimary,
     },
+    inputIcon: {},
```
Comment (HaakonSvane) [inline packages/components/src/components/TextField/TextField.tsx:97]: Can be removed

**Review by HaakonSvane — DISMISSED** [review]
👌 🤠

**Review by HaakonSvane — APPROVED** [review]
Henning, after a thorough examination of the constituent elements of your submitted pull request, it has been ascertained that said request conforms to the prescribed guidelines and fulfills the requisite criteria for approval; consequently, it has been authorized for merging into the main codebase.

**Pull request merged**